dataflux_pytorch/benchmark/checkpointing/multinode/train.py

```diff
@@ -97,14 +117,24 @@ def main(ckpt_dir_path: str, ckpt_restore_path: str = ""):
     avg_save_time = (end - start) / num_save_calls
     num_load_calls = int(os.environ.get("NUM_LOAD_CALLS", 3))
     load_checkpoint_times = []
+    if args.save_only:
+        print("Skipping loads because you set --save_only")
+        num_load_calls = 0
+        load_checkpoint_times = [0]
+    if args.strategy == FSDP_STRATEGY and args.load_only:
```

Comment:
Does there need to be some other behavior if you set load_only without FSDP_STRATEGY?

Reply:
Added a function named `validate` that throws an error when things aren't set as expected. Update this conditional accordingly. Could you take another look?